docs/content/usage/developer-workflow.md

```diff
@@ -159,9 +164,9 @@ specifies the function name, where the code lives, and associates the
 function with the python environment:
 
 ```bash
-$ fission function create --spec --name calc-form --env python --src form.py --entrypoint form.main
+$ fission function create --spec --name calc-form --env python --src "form/*" --entrypoint form.main
```

Comment:
Looks like it only works with CLI after we ship the latest release. (Archive package user experience)

Reply:
And wondering why use `--src` here? Why not use `--deploy`?